server/models/User.js

```diff
@@ -351,7 +351,8 @@ export default (Sequelize, DataTypes) => {
 
         let update = false;
         const allowedFields = 
-          [ 'firstName',
+          [ 'username',
```

Comment:
What's going to happen if there's a duplicate slug in groups table that already exists?